app/components/appMainModule/localePageModule/upcomingElectionsList/UpcomingElectionsListController.js

```diff
@@ -40,8 +40,8 @@
       var ballotListItem = {};
       ballotListItem.id = contestObject.id;
       ballotListItem.type = contestObject.contest_type.toLowerCase();
-      ballotListItem.linkTitle = contestObject.name;
-      ballotListItem.linkUrl = $filter('slugify')(contestObject.name);
+      ballotListItem.linkTitle = ballotListItem.type == 'office' ? contestObject.name : 'Measure: ' + contestObject.number + ' ' + contestObject.title;
```

Comment:
would you mind making this a separate function that returns the value necessary and is called on `ballotListItem.linkTitle`?  Since the data is not always the cleanest coming back, it will be easier to test where the problems are if things are divided up into smaller functions